notebooks/time-to-merge-prediction/thoth-station/thoth_station_feature_engg.ipynb

```diff
@@ -0,0 +1,5596 @@
+{
```

Comment:
<p>Any idea why almost all of these are NaNs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/496/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='126'/>

Reply:
<p>That might be because there is no correlation between the filetypes and the ttm variable</p>